akka-stream-tests/src/test/resources/reference.conf

```diff
@@ -2,7 +2,6 @@ akka {
   loggers = ["akka.testkit.TestEventListener"]
   actor {
     serialize-creators = on
-    serialize-messages = on
     default-dispatcher.throughput = 1 // Amplify the effects of fuzzing
```

Comment:
yes, let's remove `serialize-creators` also (that is only needed for remote deployment)